tests/ctst/features/put-object.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Put Object operations
+
+    @2.6.0
```

Comment:
```suggestion
    @2.12.0
```

The rationale is that we have a logic in ctst able to auto run or skip tests based on the version tag. The benefit is that we can decide to have the same test suite on all (maintained) branches to reduce the integration branches conflicts, by keeping the same implementation, and reuse it in other repos (like artesca) later if needed. So if we add a test starting 2.12 it's better to set 2.12 as the version tag

See https://github.com/scality/cli-testing/blob/main/hooks/versionTags.ts